internal/chezmoi/sourcestate.go

```diff
@@ -636,9 +637,18 @@ func (s *SourceState) Encryption() Encryption {
 	return s.encryption
 }
 
+var tmpl *template.Template
+
+func execTemplateTemplateFunc(name string, data interface{}) (string, error) {
+	buf := &bytes.Buffer{}
+	err := tmpl.ExecuteTemplate(buf, name, data)
```

Comment:
The main problem is that `execTemplateTemplateFunc` needs to have access to the same `tmpl` object in use in the function below. That's the reason I could not put it in the `templatefuncs.go`, but maybe there is a way to do it better that I could not think of (I'm Go beginner :P).